whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -90,7 +90,7 @@ public void verifyAndSetAsUnsubscribed(final EmailMessageInfo message) {
         }
 
         LOGGER.debug("Unsubscribe message-id {} email {}", message.messageId(), unsubscribeRequestEmail);
-        emailStatusDao.createEmailStatus(unsubscribeRequestEmail, EmailStatusType.UNSUBSCRIBE);
+        emailStatusDao.createEmailStatus(unsubscribeRequestEmail, EmailStatusType.UNSUBSCRIBE, null);
```

Comment:
May as well store the message blob for unsubscribe also?

Reply:
Why? I don't think we will need it for anything

Reply:
Good point, as discussed we don't need unsubscribed messages, so leave it NULL